README.markdown

```diff
@@ -158,6 +155,21 @@ To use a mirror of the iojs binaries, set `$NVM_IOJS_ORG_MIRROR`:
 
 `nvm use` will not, by default, create a "current" symlink. Set `$NVM_SYMLINK_CURRENT` to "true" to enable this behavior, which is sometimes useful for IDEs.
 
+### .nvmrc
+
+You can create a `.nvmrc` file containing version number in the project root directory (or any parent directory).
+`nvm use`, `nvm install`, `nvm exec`, `nvm run`, and `nvm which` will all respect an `.nvmrc` file when a version is not supplied.
+
+For example, to make NVM default to version 5 for the current directory:
```

Comment:
Fair point. I turned it to 5.9 as that's what I've currently got installed, and is actually a realistic version to lock to. Agreed?